docker/README.md

```diff
@@ -150,3 +150,12 @@ COPY . /opt/iotajson/
 ```
 
 Full instructions can be found within the `Dockerfile` itself.
+
+### Docker Secrets
+
+As an alternative to passing sensitive information via environment variables, `_FILE` may be appended to some sensitive environment variables, causing the initialization script to load the values for those variables from files present in the container. In particular, this can be used to load passwords from Docker secrets stored in `/run/secrets/<secret_name>` files.
+
```

Comment:
I'd suggest to include an example here, in the same way Cygnus dockerhub README.md (https://hub.docker.com/r/telefonicaiot/fiware-cygnus/) does.

Reply:
Fixed with https://github.com/telefonicaid/iotagent-json/pull/380/commits/399c3fbda91e1b11cec196698faf4c457ffe8705

Reply:
I have looked to and it seems line "Currently, this the `_FILE` suffix is supported for:" is duplicated. Please remove it at line 159.

Reply:
Fixed with https://github.com/telefonicaid/iotagent-json/pull/380/commits/ab8ded7ad6ea497c37bd3632425b675d3a8a550b